### DISCUSSION TITLE
istoreos旁路由模式DNS测试

### DISCUSSION THREAD

**Discussion by jesse26wong** [istoreos/istoreos#791] [General]
不懂就问：刚安装的istoreos，装了一个blid-ding插件就没装其他插件了。用设置向导设好旁路由模式后，特意将DNS转发到一个错误ip:6.6.6.6后，登录终端用ding @127.0.0.1 -p 53 baidu.com为什么用还能得到DNS解析？？
![Screenshot_20230521_094158_com huawei browser](https://github.com/istoreos/istoreos/assets/133888713/25bebc81-d076-46d5-aee2-97e1109b904d)
![Screenshot_20230521_094243](https://github.com/istoreos/istoreos/assets/133888713/fd52bdba-14ae-4698-83e8-4c50be2d44c5)

**Comment by jjm2473** [comment]
因为你安装的插件有可能在iptables里劫持53端口，自己看防火墙状态里面有没有53就知道了
